marketplaceItems/soloPoint/appResources/appData/resources/js/soloPointTablet.js

```diff
@@ -0,0 +1,130 @@
+/*
+    Solo Point
+    Created by Milad Nazeri on 2019-01-07
+    Copyright 2016 High Fidelity, Inc.
+
+    Distributed under the Apache License, Version 2.0.
+    See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+    Point to solo someone to hear them better in a crowd!
+    TABLET JS
+    
+*/
+
+
+// *************************************
+// START UTILITY FUNCTIONS
+// *************************************
+// #region Utilty
+
+var debug = true;
+var FALSE = "false";
```

Comment:
That is for the log utility so that if I want something to log, I can just leave that section blank
like log("message", object);
the first line in the logger does this:
    if (!debug || enabled === FALSE) {
        return;
    }

if enabled where !enabled or typeof 'undefined', i would always have to put a true or false.

Reply:
nevermind, realized a better way to do that without needing that at all.  Separating out to a utility module as well.

Reply:
OK